taprpc/taprootassets.proto

```diff
@@ -128,6 +128,13 @@ service TaprootAssets {
     */
     rpc SubscribeSendEvents (SubscribeSendEventsRequest)
         returns (stream SendEvent);
+
+    /*
+    FundChannel initiates the channel funding negotiation with a peer for the
+    creation of a channel that contains a specified amount of assets of a given
+    asset.
+    */
+    rpc FundChannel (FundChannelRequest) returns (FundChannelResponse);
```

Comment:
It would be nice if we could keep all channel specific RPC endpoints to a single channel RPC package.